client/signup/steps/jpo-summary/index.jsx

```diff
@@ -159,11 +159,11 @@ class JPOSummaryStep extends React.Component {
 											? null
 											: <li><a href={ `${ pathPrefix }jpo-contact-form` }>{ translate( 'Contact Us Form' ) }</a></li>
 									}
-									{ ! connectionToJetpackComplete ? <li>{ translate( 'Connection to Jetpack' ) }</li> : null }
-									<li>{ translate( 'Choose a Theme' ) }</li>
-									<li>{ translate( 'Add a Site Address' ) }</li>
-									<li>{ translate( 'Add a Store' ) }</li>
-									<li>{ translate( 'Start a Blog' ) }</li>
+									{ ! connectionToJetpackComplete ? <li>{ translate( 'Jetpack Connection' ) }</li> : null }
+									<li><a href="#">{ translate( 'Choose a Theme' ) }</a></li>
+									<li><a href="#">{ translate( 'Add a Site Address' ) }</a></li>
```

Comment:
:disappointed: 1 existing translations will be lost with this change.
If the string must change, the following option has already been translated **85** times:
`translate( 'Site Address' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).